### PR TITLE
Add CI workflow for the project

### LINKED ISSUE #2: Add a Github Actions to perform a CI
We need a Github Actions to perform a CI pipeline, for now is not necessary to publish the Docker image just building.

### PR DESCRIPTION
**Pull request by Amnish04**
Hi @mcruzdev,
I just added a `ci` workflow that checks for linting errors, builds the application, and runs the test script (commented out for now as there are no tests).

Please review my changes, and let me know if something needs to be changed before it the PR can be merged.

This fixes #2 

**Note:** The build process if failing right now due to various `typescript specific` issues throughout the project. We should see a green check once those are fixed.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      with:
+        node-version: 14 # You can specify the Node.js version you need.
+
+    - name: Install dependencies
+      run: npm install
+
+    - name: Build
+      run: npm run build
+
+    # No tests yet
```
Comment (mcruzdev) [inline .github/workflows/ci.yml:48]: I think that we can remove this comment, I will create an issue to add tests, TY!

**Review by mcruzdev — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Amnish04) [reply .github/workflows/ci.yml]: Just removed tests from ci pipeline as requested

Comment (Amnish04) [reply .github/workflows/ci.yml]: Just removed tests from ci pipeline as requested

**Comment by mcruzdev** [comment]
Thank you @Amnish04!

**Pull request merged**